lib/ssl/src/ssl_cipher.erl

```diff
@@ -1079,7 +1076,7 @@ filter_suites_signature(ecdsa, Ciphers, Version) ->
 %% extension.  The names DH_DSS, DH_RSA, ECDH_ECDSA, and ECDH_RSA are
 %% historical.
 %% Note: DH_DSS and DH_RSA is not supported
-rsa_signed({3,N}) when N >= 3 ->
+rsa_signed(?'TLS-1.X'=Version) when Version >= ?'TLS-1.2' ->
```

Comment:
This looks wrong! The functions will only be called when version is lower then TLS-1.2!

Reply:
ok, I am not sure I fully understand what is implied :)
Do you mean that I should `Version =< ?'TLS-1.2'? If I were to do this, then the next function clause for `rsa_signed(_) ->` would never be executed, correct?

Reply:
I am saying that the firs clause is dead code and we could remove the Version argument to the function.